native/src/modules/endpoint/sagas/fetchResourceCache.js

```diff
@@ -72,7 +72,7 @@ export default function* fetchResourceCache(
       // TODO: we might remember which files have failed to retry later
       // (internet connection of client could have failed)
       const message = createErrorMessage(failureResults)
-      console.warn(message)
+      console.log(message)
```

Comment:
What do you think about this, especially @maxammann? I think it is really annoying to have these warnings that a file could not be downloaded all the time during development. But if you think I should keep it I'll revert this.

Reply:
also think it is annoying, but do not know if we should keep it.

Reply:
We can remove it if it bugs you. Never had more than one message.